griptape/memory/structure/summary_conversation_memory.py

```diff
@@ -16,17 +13,27 @@
 @define
 class SummaryConversationMemory(ConversationMemory):
     offset: int = field(default=1, kw_only=True, metadata={"serializable": True})
-    prompt_driver: BasePromptDriver = field(
-        default=Factory(lambda: OpenAiChatPromptDriver(model=OpenAiTokenizer.DEFAULT_OPENAI_GPT_3_CHAT_MODEL)),
-        kw_only=True,
-    )
+    _prompt_driver: BasePromptDriver = field(kw_only=True, default=None, alias="prompt_driver")
```

Comment:
nit: these aliases that restate the field name without an underscore are redundant and mirror the default behavior of `attrs`:

>    :param str | None alias: Override this attribute's parameter name in the
        generated ``__init__`` method. If left `None`, default to ``name``
        stripped of leading underscores. See `private-attributes`.

Reply:
Ah yeah this is just to satisfy pyright. Without it, pyright will yell that `prompt_driver` is not a field.